### PR TITLE
Add the Extract the log level field section

### PR DESCRIPTION
**Pull request by mdbirnstiehl**
This PR closes [Issue 3121](https://github.com/elastic/observability-docs/issues/3121) and adds the Extract the `log.level` field to the Stream a log file documentation.

### CODE REVIEW

File: docs/en/observability/logs-stream.asciidoc [hunk 1]
```diff
@@ -389,4 +394,180 @@ Check the following common issues for possible solutions:
...
+
+The following command queries your documents based on the log level keyword and only shows those with `WARN` or `ERROR` log levels: 
+
+[source,console]
+----
+GET logs-example-default/_search
+{
+  "query": {
+    "terms": {
+      "log.level.keyword": ["WARN", "ERROR"]
```
Comment (ruflin) [inline docs/en/observability/logs-stream.asciidoc:530]: This looks off. It should be `log.level`. I think the reason here is that in the initial template you have not included the ECS base template. @eyalkoren can point you to which templates are needed to make sure things work as expected.

File: docs/en/observability/logs-stream.asciidoc [hunk 2]
```diff
@@ -389,4 +394,180 @@ Check the following common issues for possible solutions:
...
+          "log": {
+            "level": "ERROR"
+          },
+          "@timestamp": "2023-08-08T13:45:14.003Z"
+        }
+      }
+    ]
+  }
+}
+----
```
Comment (ruflin) [inline docs/en/observability/logs-stream.asciidoc:573]: Follow up request (out of scope of this PR): Show the user how to do a terms agg on the log.level field to see how many docs there are for a certain time period.

**Comment by ruflin** [comment]
Overall direction LGTM, left one comment around mappings which looks off. As usual, lets get this in quickly and then iterate on it.

**Review by bmorelli25 — APPROVED** [review]
🚢

**Pull request merged**